react/src/util/address.ts

```diff
@@ -23,18 +23,18 @@ export const isValidXecAddress = (address: string): boolean => {
   }
 };
 
-export const validCurrencyType = (address: string): currency => {
+export const getCurrencyTypeFromAddress = (address: string): currency => {
   if (isValidCashAddress(address)) {
     return 'BCH';
   } else if (isValidXecAddress(address)) {
     return 'XEC';
   } else {
-    return 'GBP';
+    throw new Error();
```

Comment:
Throw in a descriptive string or something for now for the error instance.

Reply:
I don't think this is resolved yet. Add some parameter like `new Error( 'Invalid address' )`

Reply:
you are looking at an old commit